compute/instances.go

```diff
@@ -961,17 +984,14 @@ func (api *_Instance) toNative() (*Instance, error) {
 
 // toTags() injects its state information into a Tags map suitable for use to
 // submit an API call to the vmapi machine endpoint
-func (mcns *InstanceCNS) toTags(m map[string]interface{}) {
-	if mcns.Disable != nil {
-		s := fmt.Sprintf("%t", mcns.Disable)
-		m[CNSTagDisable] = &s
+func (cns *InstanceCNS) toTags(m map[string]interface{}) {
+	if cns.Disable {
+		m[CNSTagDisable] = cns.Disable
```

Comment:
A `bool` or a "string masquerading as a `bool`" ?  Regardless, this may be worth calling out as a comment in the code:

> // NOTE(justinwr@): The API requires the CNSTagDisable attribute to be a boolean, not a bool string.

That comment probably needs to chase into a `Validator()` function, actually.